programs/mango-v4/src/accounts_ix/openbook_v2_create_open_orders.rs

```diff
@@ -0,0 +1,54 @@
+use anchor_lang::prelude::*;
+
+use crate::error::*;
+use crate::state::*;
+use openbook_v2::{
+    program::OpenbookV2,
+    state::{Market, OpenOrdersAccount},
+};
+
+#[derive(Accounts)]
+#[instruction(account_num: u32, open_orders_count: u8)]
```

Comment:
account_num may be confused with the account_num passed to account_create. Maybe open_orders_account_num? Looks like the open_orders_count doesn't need to be seen here.